server/src/db/migrations/20230504135943_addImageUrlColumn.cjs

```diff
@@ -7,7 +7,7 @@
  */
 exports.up = async (knex) => {
     return knex.schema.alterTable("websites", (table) => {
-        table.string("imageUrl").notNullable().defaultTo(null)
+        table.string("imageUrl").notNullable().defaultTo("")
```

Comment:
i believe i saw this in the other PR. Doesnt make sense to default this to an empty string if its required.
remove the `defaultTo` altogether